operator/hack/common.sh

```diff
@@ -123,6 +159,9 @@ function nurse_deployment_until_available() {
     "rhacs-operator.v${version_tag}" --type json \
     -p '[ { "op": "add", "path": "/spec/install/spec/deployments/0/spec/template/spec/imagePullSecrets", "value": [{"name": "'"${pull_secret}"'"}] } ]'
 
+  retry 30 10 kubectl -n "${operator_ns}" patch deploy rhacs-operator-controller-manager --type json \
```

Comment:
```suggestion
  log "Patching image pull secret also into rhacs-operator-controller-manager to speed things up."
  retry 30 10 kubectl -n "${operator_ns}" patch deploy rhacs-operator-controller-manager --type json \
```
Did I get the intention right?

Reply:
yes, exactly.